var/spack/repos/builtin/packages/py-ilmbase/package.py

```diff
@@ -0,0 +1,31 @@
+# Copyright 2013-2021 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack import *
+
+
+class PyIlmbase(AutotoolsPackage):
+    """The PyIlmBase libraries provides python bindings for the IlmBase libraries."""
+
+    homepage = "https://github.com/AcademySoftwareFoundation/openexr/tree/v2.3.0/PyIlmBase"
+    url      = "https://github.com/AcademySoftwareFoundation/openexr/releases/download/v2.3.0/pyilmbase-2.3.0.tar.gz"
```

Comment:
How does pyilmbase relate to OpenEXR? This asset seems to only be available for older versions of OpenEXR. Is it deprecated, or are the features now part of OpenEXR?

Reply:
I'm not super happy about this package. It's an [optional dependency](https://github.com/alembic/alembic/blob/1.7.16/README.txt#L23) of #22223 which itself is a [dependency of Blender](https://developer.blender.org/diffusion/B/browse/blender-v2.92-release/build_files/build_environment/cmake/versions.cmake$81) (coming down the pipe soon-ish). I don't know if I need the python binding for alembic for blender/our users, but I'm trying to err on the side of caution for that.

This all gets more annoying in that pyilmbase is still a sub project of another project in newer releases. It just [moves to Imath](https://github.com/AcademySoftwareFoundation/openexr/blame/74d59367e6dfc829b255463a780124465e940b9a/CHANGES.md#L51) instead. Unfortionately, from what I can see, only beta versions of the software involved have been released that reflect this change.